docs/sources/write/style-guide/useful-link-text/index.md

```diff
@@ -0,0 +1,57 @@
+---
+aliases:
+  - /docs/writers-toolkit/write/useful-link-text/
+title: Write useful link text
+description: Learn how to write useful link text.
+---
+
+# Write useful link text
+
+Link text is important for search engines and users, and is critical for accessibility.
+It's the visible, clickable text in an HTML hyperlink.
+
+The primary purpose of link text is to give users a short summary of the link destination content.
+
+## Summary
+
+When writing link text, pay attention to words, quantity of links, and the length of link text.
+
+### Words
+
+- Make link text obviously reflect what the next page is about.
+- Don't use words such as _Click here_, _More_, and _Read more_ as standalone link text, because they can be confusing when a screen reader reads them out of context.
+
+- When linking to a GitHub repository, include `repository` in the sentence text or link text.
+
+  For example:
+
+  - To learn more, go to the `[faro-web-sdk]()` repository.
+  - To learn more about Grafana Faro, go to its `[repository]()`.
+
+- When linking from outside of a documentation page, include `documentation` in the text.
```

Comment:
Do we ever link from outside a doc page? On blogs maybe? Or?

Reply:
Yeah, I believe that's the intention of this sentence. If you are writing a blog or product page link to the documentation.